### PR TITLE
Add clarifying notes on install directions

### PR DESCRIPTION
**Pull request by lorenanicole**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ much difficulty to most other Unix-like operating systems.
    npm.  In Terminal, run:
 
         $ sudo npm install npm -g
-        $ sudo np
+        $ sudo npm
```
Comment (lorenanicole) [inline INSTALL.md]: @cecilia-donnelly I'm not sure if this is correct, but thought it is ...

File: INSTALL.md [hunk 2]
```diff
@@ -48,28 +48,26 @@ didn't, you might want to add one:
 5. Set up the live config files.  Note that there are multiple files to
 be edited here.
 
-  1. Do `cp config.js.tmpl config.js`, then edit the latter.
+  1. Do `cp config.js.tmpl config.js` and edit the `config.js` file:
```
Comment (lorenanicole) [inline INSTALL.md]: @cecilia-donnelly default here to more specific language (as see below)

**Comment by lorenanicole** [comment]
@cecilia-donnelly the 'start forever' step didn't work for me. But everything else worked final, I can run a local instance at least! Unclear if this should be the branch I issue install changes to so please close if incorrect.

File: INSTALL.md (hunk 1 above)
Comment (cecilia-donnelly) [inline INSTALL.md]: Actually I think we can just remove that line.  It looks like it was added accidentally.

**Pull request merged**